### DISCUSSION TITLE
Way to change already racked devices from half to full depth?

### DISCUSSION THREAD

**Discussion by bradbendy** [netbox-community/netbox#12758] [General]
I don't see a way to do this but figured I would ask. We created device templates that were half depth instead of full, have updated those so all new ones are correct. Any way to change the devices already created to full depth?

Thanks

**Comment by jeremystretch** [comment]
Depth is a property of the device type, rather than the individual devices, so if you've already updated your device types you shouldn't need to make any further changes.
